pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -868,23 +850,56 @@ def list(self, request, *args, **kwargs):
         """
         Returns paginated CollectionVersions list.
         """
+
+        # there are too many potential permutations of query params to be able
+        # to cache all of them, so this will only cache queries that stick to
+        # limit/offset params, which the ansible-galaxy client uses
```

Comment:
What are all the endpoints that the ansible-client uses?

Reply:
- `index/<namespace>/<name>/versions/`
- `index/<namespace>/<name>/versions/<version>/`
- `artifacts/collections/<path>/<filename>/`